arduino_address: 192.168.0.10 
arduino_port: 5060

odom_frame: odom
odom_topic: odom
odom_rate: 20.0

cmd_vel_topic: cmd_vel
map_frame: map
chair_frame: base_footprint
base_frame: base_footprint

linear_velocity_scale: 0.25
angular_velocity_scale: 0.25

hector_trajectory_server:

  source_frame_name: base_footprint